iroha_network/src/lib.rs

```diff
@@ -141,6 +141,14 @@ pub struct Connection {
     tcp_stream: SyncTcpStream,
 }
 
+/// `Receipt` should be used by [Consumers](https://github.com/cloudevents/spec/blob/v1.0/spec.md#consumer)
+/// to notify [Source](https://github.com/cloudevents/spec/blob/v1.0/spec.md#source) about
+/// [Message](https://github.com/cloudevents/spec/blob/v1.0/spec.md#message) consumption.
+#[derive(Io, Encode, Decode)]
+pub enum Receipt {
+    Ok,
+}
```

Comment:
I think `Receipt` should contain the hash of the received message or something like that. Maybe add a TODO here?

Reply:
https://wiki.hyperledger.org/display/iroha/Maintenance+Endpoint?focusedCommentId=31204541#comment-31204541 made a comment in RFC, let's discuss it there because looks like it's a high-level design question